### PR TITLE
Break types into their own files.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
This is a followup PR to: https://github.com/dotnet/roslyn/pull/19970

Nothing has changed here except for moving types into their own files.

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
integration test failures

**Comment by CyrusNajmabadi** [comment]
retest windows_debug_vs-integration_prtest please

**Comment by CyrusNajmabadi** [comment]
retest windows_release_vs-integration_prtest please

**Comment by CyrusNajmabadi** [comment]
retest windows_debug_vs-integration_prtest please
retest windows_release_vs-integration_prtest please

**Comment by CyrusNajmabadi** [comment]
retest windows_debug_vs-integration_prtest please

**Comment by CyrusNajmabadi** [comment]
retest windows_release_vs-integration_prtest please

**Comment by CyrusNajmabadi** [comment]
Telling @MattGertz This is part of OOP, but is literally nothing but breaking out types into their own files.  This was done as a separate PR ot make previous PRs easier to review.  i.e. originally we had a single type.  Then in a previous PR it was broken into two PRs.  BUt they were kept in the same file to make the diff very easy to understand.  This PR now moves those broken out types into their own files as is our general style for the IDE.

**Comment by CyrusNajmabadi** [comment]
retest windows_release_vs-integration_prtest please

**Comment by CyrusNajmabadi** [comment]
retest windows_debug_vs-integration_prtest please

**Comment by CyrusNajmabadi** [comment]
retest windows_debug_unit64_prtest please

File: src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs [hunk 1]
```diff
@@ -185,6 +185,11 @@ public class P2 { }");
         public void GFUFuzzyMatchAfterRenameTracking()
         {
             SetUpEditor(@"
+namespace N
```
Comment (Pilchie) [inline src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs]: These last few files seam out of place...

**Pull request merged**